torchvision/prototype/datasets/_builtin/pcam.py

```diff
@@ -40,15 +44,25 @@ def __iter__(self) -> Iterator[Tuple[str, io.IOBase]]:
 _Resource = namedtuple("_Resource", ("file_name", "gdrive_id", "sha256"))
 
 
-class PCAM(Dataset):
-    def _make_info(self) -> DatasetInfo:
-        return DatasetInfo(
-            "pcam",
-            homepage="https://github.com/basveeling/pcam",
-            categories=2,
-            valid_options=dict(split=("train", "test", "val")),
-            dependencies=["h5py"],
-        )
+@register_info(NAME)
+def _info() -> Dict[str, Any]:
+    return dict(
+        categories=[0, 1],
+        dependencies=["h5py"],
```

Comment:
Do we need this in the static info? Or better, can someone actually do something with this information at runtime? In #5473 I added a [`dependencies` parameter to `Dataset2.__init__`](https://github.com/pytorch/vision/pull/5473/files#diff-74f5dec81cf4038124e53c3db44c91281f1c60d6f036574a07c3dc2ce30d65ed) similar to what the current `DatasetInfo` is doing:

https://github.com/pytorch/vision/blob/63576c9f46d9c6f471db789d41149cdb6fea435b/torchvision/prototype/datasets/utils/_dataset.py#L29

Maybe we can wait for the other PR to be merged and adopt this change here?